_studio/mfx_lib/shared/src/mfx_vpx_dec_common.cpp

```diff
@@ -331,11 +331,8 @@ namespace MFX_VPX_Utility
             return false;
 
         // both zero or not zero
-        if ((p_in->mfx.FrameInfo.AspectRatioW || p_in->mfx.FrameInfo.AspectRatioH) && !(p_in->mfx.FrameInfo.AspectRatioW && p_in->mfx.FrameInfo.AspectRatioH))
-            return false;
-        
-        if ((p_in->mfx.FrameInfo.AspectRatioW && p_in->mfx.FrameInfo.AspectRatioW != 1) ||
-            (p_in->mfx.FrameInfo.AspectRatioH && p_in->mfx.FrameInfo.AspectRatioH != 1))
+        if ((p_in->mfx.FrameInfo.AspectRatioW != 0 && p_in->mfx.FrameInfo.AspectRatioH == 0) ||
+            (p_in->mfx.FrameInfo.AspectRatioW == 0 && p_in->mfx.FrameInfo.AspectRatioH != 0))
```

Comment:
Simplify if() to if ((p_in->mfx.FrameInfo.AspectRatioW == 0) != (p_in->mfx.FrameInfo.AspectRatioH == 0))